client/my-sites/domains/domain-management/list/item.jsx

```diff
@@ -159,29 +159,30 @@ class ListItem extends React.PureComponent {
 
 		if ( domain.expired ) {
 			return (
-				<Notice isCompact status="is-error" icon="spam">
-					{ translate( 'Expired %(timeSinceExpiry)s', {
+				<DomainNotice
+					status="alert"
+					text={ translate( 'Expired %(timeSinceExpiry)s', {
 						args: {
 							timeSinceExpiry: moment( domain.expiry ).fromNow(),
 						},
 						context:
-							'timeSinceExpiry is of the form "[number] [time-period] ago" i.e. "3 days ago"',
+							'timeSinceExpiry is of the form "[number] [time-period] ago" e.g. "3 days ago"',
 					} ) }
-				</Notice>
+				/>
 			);
 		}
 
 		if ( domain.expiry && moment( domain.expiry ) < this.props.moment().add( 30, 'days' ) ) {
 			return (
-				<Notice isCompact status="is-error" icon="spam">
-					{ translate( 'Expires %(timeUntilExpiry)s', {
+				<DomainNotice
+					status="warning"
+					text={ translate( 'Expires %(timeUntilExpiry)s', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.